LiveStream/Service/LiveStreamService.swift

```diff
@@ -20,7 +20,7 @@ public struct LiveStreamService: LiveStreamServiceProtocol {
         .flatMap { "?uid=\($0)" }
         .coalesceWith("")
 
-      let urlString = "\(Secrets.LiveStreams.endpoint)/\(eventId)\(uidString)"
+      let urlString = "\(Secrets.LiveStreams.Api.base)/ksr-ios-streams/\(eventId)\(uidString)"
```

Comment:
feel like we shouldn't be hardcoding `ksr-ios-streams` since it's used in a few places. at the very least, can be a `let` in this file.